packages/gafl-webapp-service/src/server.js

```diff
@@ -61,34 +63,79 @@ const createServer = options => {
 // This is a hash of the inline script at line 31 of the GDS template. It is added to the CSP to except the in-line
 // script. It needs the quotes.
 const scriptHash = "'sha256-+6WnXIl4mbFTCARd8N3COQmT3bJJmo32N8q8ZSQAIcU='"
-const plugIns = [
```

Comment:
This is getting rather cumbersome - we ought to think about extracting it to another file at some point - not now